File: src/main/java/org/example/eventhub/dto/UserResponseLong.java
package org.example.eventhub.dto;

import org.example.eventhub.enums.Role;

import java.time.LocalDateTime;

public record UserResponseLong(
        Long id,
        String username,
        String email,
        Role role,
        LocalDateTime createdAt,
        long countOrganizedEvents
) {
}

File: src/main/java/org/example/eventhub/dto/user/UserResponseLong.kt
package org.example.eventhub.dto.user

import org.example.eventhub.enums.Role
import java.time.LocalDateTime

data class UserResponseLong(
    val id: Long?,
    val username: String,
    val email: String,
    val role: Role,
    val createdAt: LocalDateTime?,
    val countOrganizedEvents: Long
)
